pkgs/tools/inputmethods/evscript/default.nix

```diff
@@ -2,19 +2,19 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "evscript";
-  version = "git-${builtins.substring 0 7 src.rev}";
+  version = "unstable-2021-06-16";
 
   src = fetchFromGitHub {
     owner = "myfreeweb";
     repo = pname;
-    rev = "47f86f0d15add2af785ea1ff47f24d130026d1b4";
-    sha256 = "1xm8297k0d8d0aq7fxgmibr4qva4d02cb6gnnlzq77jcmnknxv14";
+    rev = "410603abf4810320bf79bde90cc85367b71a8b12";
```

Comment:
It is not allowed to directly fetch commits from a fork. Please fetch from the main repo and add your changes as patch(es).